Libraries/LibGUI/GItemView.cpp

```diff
@@ -139,6 +142,13 @@ void GItemView::mouseup_event(GMouseEvent& event)
         update();
         return;
     }
+    int item_index = item_at_event_position(event.position());
+    int numselected = selection().size();
```

Comment:
Same issue here, this local variable doesn't seem to add anything other than giving an alternate name to `selection().size()`